### PR TITLE
[Remove Spring extender usages] LPS 159283

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang @tinatian,

This PR is to remove spring extender usage in `akismet-service`, thanks!
https://issues.liferay.com/browse/LPS-159283

### CODE REVIEW

**Review by dantewang — APPROVED** [review]
@zhangyan0701 Could you review the test results when they finish? Thanks!

**Comment by JanisZhang** [comment]
Hi @catarineaguiar, Could you please review the test results? Thanks!

**Comment by catarineaguiar** [comment]
@JanisZhang no regressions found.

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

File: modules/dxp/apps/akismet/akismet-service/src/main/java/com/liferay/akismet/service/impl/AkismetEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -16,15 +16,23 @@
...
 import java.util.Date;
 
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
 /**
  * @author Jamie Sammons
  */
+@Component(
+	enabled = false,
```
[the remark this answers is not in the document]
Comment (dantewang) [reply modules/dxp/apps/akismet/akismet-service/src/main/java/com/liferay/akismet/service/impl/AkismetEntryLocalServiceImpl.java:32]: @JanisZhang I think you should exclude enabled=false here.

[the remark this answers is not in the document]
Comment (JanisZhang) [reply modules/dxp/apps/akismet/akismet-service/src/main/java/com/liferay/akismet/service/impl/AkismetEntryLocalServiceImpl.java:32]: @dantewang Got It.

**Comment by JanisZhang** [comment]
@dantewang force-pushed based on the comment. Thanks!

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:test

**Comment by dantewang** [comment]
@tinatian Issue corrected, tests re-triggered.

@catarineaguiar Could you review the test results when they finish? Thanks!

**Comment by catarineaguiar** [comment]
not possible to analyze default results, because most tests were aborted. But according to relevant results no regression was found.

**Comment by vicnate5** [comment]
No regressions in the results we did get here. Passing QA review.

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina.

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/121153

**Pull request closed without merging**